### PR TITLE
Add info for 5 universities to csv and increase field lengths

### PR DESCRIPTION
**Pull request by aparajitaghimire**

### CODE REVIEW

**Review by KevinLi3 — APPROVED** [review]
LGTM!

File: README.md [hunk 1]
```diff
@@ -23,18 +23,20 @@ When a migration is made ensure that you perform the following steps before runn
...
-2. mysql> quit 
-Reconnect to the sql shell using: 
+2. mysql> quit
+   Reconnect to the sql shell using:
 3. mysql --local-infile=1 -u root -p
-Run the following commands to seed the University and UWCourse tables:
+   Run the following commands to seed the University and UWCourse tables:
 4. mysql> use xchange;
-5. mysql> `LOAD DATA LOCAL INFILE '/path/to/xchange/api/seed_university.csv' INTO TABLE university FIELDS TERMINATED BY '|' LINES TERMINATED BY '\n' (id, name, competition, languages, program, terms, location);`
+5. mysql> `LOAD DATA LOCAL INFILE '/Users/aghimire/Desktop/XChange/xchange/api/scrapers/seed_university.csv' INTO TABLE university FIELDS TERMINATED BY '|' LINES TERMINATED BY '\n\n\n' (id, name, competition, languages, program, terms, location, academic_level, requirements, tuition, transcript, housing, faculties, dates, financial_support, contact, cost, cost_disclaimer, course_info, other_info, wellness);`
```
Comment (nehageorge) [inline README.md]: Path shouldnt have aghimire

**Review by nehageorge — COMMENTED** [review]
lgtm except minor chang

File: README.md (hunk 1 above)
Comment (aparajitaghimire) [reply README.md]: oh naur

**Pull request merged**